lib/crypto.ts

```diff
@@ -0,0 +1,32 @@
+import crypto from 'crypto'
+
+const ENCRYPTION_KEY = Buffer.from(process.env.ENCRYPTION_KEY || '', 'hex')
+const ALGORITHM = 'aes-256-cbc'
+const IV_LENGTH = 16
+
+if (!ENCRYPTION_KEY || ENCRYPTION_KEY.length !== 32) {
+  throw new Error('ENCRYPTION_KEY environment variable must be a 32-byte hex string')
+}
+
+export const encrypt = (text: string): string => {
+  if (!text) return text
+
+  const iv = crypto.randomBytes(IV_LENGTH)
+  const cipher = crypto.createCipheriv(ALGORITHM, ENCRYPTION_KEY, iv)
+  const encrypted = Buffer.concat([cipher.update(text, 'utf8'), cipher.final()])
+
+  return `${iv.toString('hex')}:${encrypted.toString('hex')}`
+}
+
+export const decrypt = (encryptedText: string): string => {
```

Comment:
The decrypt function doesn't handle malformed encrypted text (e.g., missing colon separator)

Could crash if database contains invalid encrypted data

Recommendation: Add try-catch and validation, something like:

```
export const decrypt = (encryptedText: string): string => {
  if (!encryptedText) return encryptedText
  
  if (!encryptedText.includes(':')) {
    throw new Error('Invalid encrypted text format')
  }
  
  try {
    const [ivHex, encryptedHex] = encryptedText.split(':')
    // ... rest of code
  } catch (error) {
    throw new Error('Failed to decrypt: ' + error.message)
  }
}
```